keras_cv/layers/preprocessing_3d/global_random_flip.py

```diff
@@ -44,11 +45,15 @@ class GlobalRandomFlipY(base_augmentation_layer_3d.BaseAugmentationLayer3D):
 
     """
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(self, axis=1, **kwargs):
```

Comment:
Instead of axis why don't we use `y=True, x=False, z=False` and raise on x and Z?  thoughts?

Reply:
sgtm -- I've called them flip_x/y/z instead though. wdyt?